### ISSUE TITLE
ml partition for classification requires no validation set

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#2899]
When partitioning data for classification, only train and test sets are required. The Classification tab and logic should not include the validation set.

**Comment by moellep** [comment]
Also part of this ticket, Classification sims should not show or use an output scalar.
